android/digitalink/app/src/main/java/com/google/mlkit/samples/vision/digitalink/InkMainActivity.java

```diff
@@ -37,7 +37,7 @@
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_main);
+        setContentView(R.layout.activity_ink_main);
```

Comment:
Same here, as 'activity_ditigal_ink_main', and also the following places.

Reply:
Of course, but I guess you mean 
`activity_digital_ink_main` instead of 
`activity_ditigal_ink_main`